notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<p>The recognition results for the LCD screen seems great for this testing image. However, it seems configurations according to the input image are critical to the final recognition results, such as POINTS, DESIGN_SHAPE, DESIGN_LAYOUT. Could you provide clear guidance on how to obtain the exact numbers for the configurations of these variables? like using what app/function to determine the exact values when given an input image? What's the exact meaning of "DESIGN_LAYOUT"? It'll greatly improve the experience for other developers to use your notebook on their own testing images.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>